### PR TITLE
add info about the Nextcloud VM

### PR DESCRIPTION
**Pull request by enoch85**
It's production ready now.

cc @nextcloud/developers @ezraholm50

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,11 @@ changed it substantially:
 
 More information how to contribute: https://nextcloud.com/contribute/
 
+## Nextcloud VM
+If you're not familiar with Linux, or simply just want to get up and running on a pre-configured system in no time - we have developed a VM that you just download. Just extract it and mount it in VMware or VirtualBox and your're all set.
```
Comment (nickvergessen) [inline README.md]: `s/ that you just download/ that you can download`

File: README.md [hunk 2]
```diff
@@ -45,6 +45,11 @@ changed it substantially:
...
+
+You can download the latest version [here](https://github.com/nextcloud/vm/releases/)
```
Comment (jancborchardt) [inline README.md]: Instead of using »here« as the link text, better: »Download the latest version of the [Nextcloud VM]«.

**Comment by jancborchardt** [comment]
@enoch85 please ideally never mention the developer group – too many people are in there. ;)
Besides good, just see my comment – awesome the VM is done!

**Comment by enoch85** [comment]
All good? :)

**Comment by MorrisJobke** [comment]
👍

**Pull request merged**